tests/test-dirs/locate-type.t/run.t

```diff
@@ -1,3 +1,8 @@
+  $ ocamlc b.ml a.ml -bin-annot -o a.cmi
```

Comment:
That `-o a.cmi` looks wrong:
```
$ ocamlc b.ml a.ml -bin-annot -o a.cmi
$ file b.cmi
b.cmi: OCaml interface file (.cmi) (Version 029)
$ file a.cmi
a.cmi: a ...opam/4.12.0/bin/ocamlrun script executable (binary data)
```
Why don't you just replace it by `-c`?

Reply:
And actually, looking at the rest of the test, you only need to compile `b.ml`, not `a.ml`.